website/templates/website/event.html

```diff
@@ -33,6 +33,14 @@ <h3 class="event-detail">Event Details</h3>
       <h3 class="event-detail">Description</h3>
       <p>{{ event.description }}</p>
 
+      {% if user.is_staff is False %}
+      <p>
+        <form method="get" action="registration">
```

Comment:
I think it is better to make this an anchor instead of using a `get` form?

Reply:
Is there a CSS class I can use to keep the button style? I was worried that an ordinary link without any styling wouldn't stand out enough on the page. Using the form was a hack to keep the button aesthetic without requiring the user to run any JavaScript with onclick.